### DISCUSSION TITLE
Return switching tracks with a swipe on the album cover

### DISCUSSION THREAD

**Discussion by netrunner-exe** [z-huang/music#996] [General]
Hello @z-huang!  I would like to ask, do you plan to return this feature?  Perhaps you are investigating a bug with the horizontal switcher or have no plans to return this feature at all?  You did a great job implementing this feature, so I would still like to bring it back again. Thank you!

**Comment by z-huang** [comment]
If I find a way to add this feature without bugs, I'll add it back.

**Reply by aayushrautela** [reply]
That would be a great addition.
